master/buildbot/status/web/hooks/github.py

```diff
@@ -100,10 +101,10 @@ def handle_push(self, payload):
         user = None
         # user = payload['pusher']['name']
         repo = payload['repository']['name']
-        repo_url = payload['repository']['url']
+        repo_url = payload['repository']['clone_url']
         # NOTE: what would be a reasonable value for project?
         # project = request.args.get('project', [''])[0]
-        project = payload['repository']['full_name']
+        project = payload['repository']['name']
```

Comment:
They were merely to match our codebase repositories and project names, which are using the `.git` urls instead of the web-facing urls (good practice I suppose) and the repo name instead of account+repo name of `full_name`. Codebases simply weren't matching because of this. This should probably be configurable instead.